src/main/java/org/mockito/plugins/MockResolver.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.plugins;
+
+/**
+ * A mock resolver offers an opportunity to resolve a mock from any instance that is
+ * provided to the {@link org.mockito.Mockito}-DSL. This mechanism can be used by
+ * frameworks that provide mocks that are implemented by Mockito but which are wrapped
+ * by other instances to enhance the proxy further.
+ */
```

Comment:
This needs to be marked as `@Incubating`

Reply:
Arghs, I missed this and just hit merge. Sorry, we can still add this, though.

Reply:
Yes let's still add it.